teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(age, specialization, **args)
+    super(age, **args)
```

Comment:
- [ ] Let us modify our initialize like this
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)